mypy/build.py

```diff
@@ -539,8 +539,6 @@ class BuildManager:
       modules:         Mapping of module ID to MypyFile (shared by the passes)
       semantic_analyzer:
                        Semantic analyzer, pass 2
```

Comment:
Is there still a pass 1, and if so why isn't it mentioned here?

Reply:
Yes, there is: https://github.com/python/mypy/blob/5253f7c0214f598bdc8b96e0e90e9dd459a96b25/mypy/build.py#L2105-L2126

It is not mentioned, because we don't use any props when running the first pass.